nexus/external-api/src/lib.rs

```diff
@@ -3088,6 +3094,87 @@ pub trait NexusExternalApi {
         rqctx: RequestContext<Self::Context>,
         params: TypedBody<params::DeviceAccessTokenRequest>,
     ) -> Result<Response<Body>, HttpError>;
+
+    // Webhooks (experimental)
+
+    /// Get the configuration for a webhook.
```

Comment:
Small note for consistency, all of the docs should use same naming convention. I think "webhook receiver" as is used below is good.
```suggestion
    /// Get the configuration for a webhook receiver.
```